### ISSUE TITLE
Completely broken with webpack 2.2

### ISSUE THREAD

**Issue by aight8** [webpack-contrib/extract-text-webpack-plugin#352]
```
,/xxx/node_modules/extract-text-webpack-plugin/loader.js?{"omit":1,"remove":true}!/xxx/node_modules/style-loader/index.js!/Users/xxxxx/xxxx/xxxx/frontend/node_modules/css-loader/index.js!/Users/xxxx/xxxx/xxxxx/node_modules/sass-loader/index.js!/Users/xxxx/jjd/xxxxx.scss doesn't export content,./src/apps/platform/content/home/style.scss
Module build failed:   Error: 
  
  - NormalModule.js:141 
    [frontend]/[webpack]/lib/NormalModule.js:141:35
  
  - LoaderRunner.js:359 
    [frontend]/[loader-runner]/lib/LoaderRunner.js:359:11
  
  - LoaderRunner.js:225 
    [frontend]/[loader-runner]/lib/LoaderRunner.js:225:18
  
  - LoaderRunner.js:138 runSyncOrAsync
    [frontend]/[loader-runner]/lib/LoaderRunner.js:138:3
  
  - LoaderRunner.js:224 iterateNormalLoaders
    [frontend]/[loader-runner]/lib/LoaderRunner.js:224:2
  
  - LoaderRunner.js:213 iterateNormalLoaders
    [frontend]/[loader-runner]/lib/LoaderRunner.js:213:10
  
  - LoaderRunner.js:228 
    [frontend]/[loader-runner]/lib/LoaderRunner.js:228:3
  
  - LoaderRunner.js:106 context.callback
    [frontend]/[loader-runner]/lib/LoaderRunner.js:106:13
  
  - index.js:289 Object.onRender
    [frontend]/[sass-loader]/index.js:289:9
  
  - async.js:2234 Object.<anonymous>
    [frontend]/[sass-loader]/[async]/dist/async.js:2234:31
```

This error is outputed for every style import and the build fail.
Using **webpack 2.2** and **extract-text-webpack-plugin beta 5**.
The configuration is correct, tried every kind of examples for v2.
I want to use css with postcss, sass - however for testing I tried without those. But the same error.

**Comment by michael-ciniawsky** [comment]
@aight8 `webpack.config.js` and one component you import styles to please

**Comment by mastertinner** [comment]
I'm having the same error using the [Angular-CLI](https://github.com/angular/angular-cli) Webpack config and importing a single `.css` file from a `.scss` file.

**Comment by michael-ciniawsky** [comment]
@mastertinner webpack.config.js and one component you import styles to please :)

Did angluar-cli already update to `extract-text-plugin` v2-rc? Could you open an issue there aswell either asking to update or to investigate further?

**Comment by aight8** [comment]
I sadly don't know how I fixed this issue, I deleted the nodes_modules and cache etc. but maybe it was a slight config change. I try to put here more infos when I know more...

**Comment by michael-ciniawsky** [comment]
@aight8 Where do yu setup postcss options, does it work without it? Could you please post the config `postcss.config.js`, `webpack.config.js`, ?`angularrc` (I don't use angular 😛)

**Comment by aight8** [comment]
angularrc I dont use either ;) The post css is basically just that:
```JS
module.exports = context => {
  return {
    plugins: [
      require('lost'), // http://lostgrid.org/docs.html
    ],
  };
};
```

The config is generated and is quite complex. I will extract the effective generated client webpack config when I'm back on this tasks.

**Comment by mastertinner** [comment]
@michael-ciniawsky, the Angular-CLI doesn't let you configure your own `webpack.config.js` but I believe that it's using this config: https://github.com/angular/angular-cli/tree/master/packages/angular-cli/models/webpack-configs. In our case, we are importing a `.scss` which contains the following line: `@import some/file.css`. That's where the problem happens.

**Comment by michael-ciniawsky** [comment]
> In our case, we are importing a .scss which contains the following line: @import some/file.css. That's where the problem happens.

@mastertinner thx, does SASS support plain `.css` @import(s) ? I will take a deeper look at the cli webpack config(s) when I have time

**Comment by mastertinner** [comment]
@michael-ciniawsky, yes it does. It has worked up until a few days ago :)

**Comment by aight8** [comment]
I realised that if I using css-loader v0.14.x it's working again. don't know if this is the problem.

**Comment by bebraw** [comment]
Can anyone confirm this? There's not enough information to debug. Ideally you should be able to isolate a small project to study. Otherwise this won't progress.

**Comment by tehnomaag** [comment]
OK, after digging through a lot of things I figured out why my build suddenly broke with same _"... .css doesn't export content"_ errors.

Turns out I accidentally ran `yarn clean` instead of `yarn run clean` (my custom script defined in package.json). This created a local `.yarnclean` file which blacklisted `.*.yml` files and thus `.svgo.yml` was removed from `css-loader > cssnano > postcss-svgo > svgo` dependency installation. This dep was triggered when I was using `ExtractTextPlugin.extract` and `css-loader` with `minimize: true` option (thus triggering internally svgo for svg-minimization, and svgo fails if it can't load its `.svgo.yml` config file).

This seems to be a known issue in yarn, see https://github.com/yarnpkg/yarn/issues/1051

Fix is to either remove `.yarnclean` or add `!.svgo.yml` in there and do a `rm -rf node_modules && yarn`.

I don't think this is an issue with `npm` based installs, since there's no cleaning feature there.
